Dockerfile

```diff
@@ -5,11 +5,12 @@ ENV PATH ${PATH}:/go/bin
 ENV GO111MODULE=on
 ENV CGO_ENABLED=0
 ENV GOBIN=/bin
+ARG BUILDARCH=amd64
 RUN rm -r /etc/vpp
 RUN go install github.com/go-delve/delve/cmd/dlv@v1.8.2
-ADD https://github.com/spiffe/spire/releases/download/v1.2.2/spire-1.2.2-linux-x86_64-glibc.tar.gz .
+ADD https://github.com/spiffe/spire/releases/download/v1.8.0/spire-1.8.0-linux-${BUILDARCH}-musl.tar.gz .
 ADD https://github.com/coredns/coredns/releases/download/v1.9.1/coredns_1.9.1_linux_amd64.tgz .
```

Comment:
Could you please double check if we need to use the `BUILDARCH` variable for `coredns` too?